react/components/atoms/input/input.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, boolean, text } from "@storybook/addon-knobs";
+
+import { Input } from "./input";
+
+storiesOf("Components/Atoms/Input", module)
+    .addDecorator(withKnobs)
+    .add("Input", () => {
+        const value = text("Value", undefined);
+        const showBorder = boolean("Show Border", true);
+        const borderColor = text("Border Color", "#e4e8f0");

```

Comment:
Move this to the upper components, and keep the input barebones

Reply:
Can you please clarify @gcandal 

Remove the props from the component and add a style prop to be used by the parent ?

Remove just from the story ?